smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -0,0 +1,184 @@
+pragma solidity 0.4.25;
+
+import './MixinDisableAndDestroy.sol';
+import './MixinApproval.sol';
+import './MixinKeys.sol';
+import './MixinLockCore.sol';
+import 'openzeppelin-solidity/contracts/utils/Address.sol';
+import 'openzeppelin-solidity/contracts/token/ERC721/IERC721Receiver.sol';
+import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+
+/**
+ * @title Mixin for the transfer-related functions needed to meet the ERC721
+ * standard.
+ * @author Nick Furfaro
+ * @dev `Mixins` are a design pattern seen in the 0x contracts.  It simply
+ * separates logically groupings of code to ease readability.
+ */
+
+contract MixinTransfer is
+  MixinApproval,
+  MixinKeys,
+  MixinLockCore {
+  using SafeMath for uint;
+  using Address for address;
+
+  // 0x150b7a02 == keccak256('onERC721Received(address,address,uint256,bytes)')
```

Comment:
For completeness maybe we should include `bytes4(keccak256(...`

Reply:
done.